src/Playwright.NUnitTest/PlaywrightTest.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+using System.Threading;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using Microsoft.Playwright;
+
+namespace Microsoft.Playwright.NUnitTest
+{
+    public class PlaywrightTest
+    {
+        private static Task<IPlaywright> playwrightTask_ = Microsoft.Playwright.Playwright.CreateAsync();
+
+        internal IPlaywright Playwright { get; private set; }
+
+        [SetUp]
+        public async Task PlaywrightSetup()
+        {
+            Playwright = await playwrightTask_;
```

Comment:
I would rather do something like 

```
Playwright = Microsoft.Playwright.Playwright.CreateAsync().GetAwaiter().GetResult();
```

Reply:
This would create more than one Playwright.